src/data/services/LmsApiService.js

```diff
@@ -196,6 +198,10 @@ class LmsApiService {
     const url = `${LmsApiService.enterpriseCustomerUrl}${enterpriseId}/enterprise_learners/`;
     return LmsApiService.apiClient().post(url, options);
   }
+
+  static createPendingEnterpriseUsers(formData) {
+    return LmsApiService.apiClient().post(this.createPendingUsersUrl, formData);
```

Comment:
Does `this` actually work with the way the class is setup?
Every other reference is `LmsApiService.function/varname`

Reply:
IT SURE SEEMS TO BUT YOU ARE RIGHT and there was some weird error elsewhere from using this and I will change it, thanks!